app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>PrivateEvents</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-eOJMYsd53ii+scO/bJGFsiCZc+5NDVN2yr8+0RDqr0Ql0h+rP48ckxlpbzKgwra6" crossorigin="anonymous">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/js/bootstrap.bundle.min.js" integrity="sha384-JEW9xMcG8R+pH31jmWH6WWP0WintQrMb4s7ZOdauHnUtxwoG2vI5DkLtS3qm9Ekf" crossorigin="anonymous"></script>
+    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+</head>
+
+<body>
+
+    <% if user_signed_in? %>
+    <%= link_to 'Logout', destroy_user_session_path, method: :delete, class:"btn btn-lg btn-danger text-white" %>
+    <%= link_to 'New Event', new_event_path, class:"btn btn-lg btn-success text-white" %>
+    <%= link_to 'My profile', user_path(current_user), class:'btn btn-lg btn-primary text-white' %>
+    <% else %>
+    <%= link_to 'Register', new_user_registration_path, class:"btn btn-lg btn-success text-white" %>
+    <%= link_to 'Login', new_user_session_path, class:"btn btn-lg btn-primary text-white" %>
+
+    <% end %>
```

Comment:
**Optional**

- [ ] Move this logic to a helper. Views are only for rendering what's made available to them from the corresponding controller. And not for generating it.